app/src/androidTest/java/com/sthoray/allright/ExploreActivityTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.sthoray.allright
+
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import org.junit.Assert.*
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class ExploreActivityTest{
+
+    @get: Rule
+    val activityRule = ActivityScenarioRule(ExploreActivity::class.java)
+
+
+    @Test
+    fun test_isActivityInView() {
+
+        //This function launches the activity and tests that it is in view
+        onView(withId(R.id.exploreActivity)).check(matches(isDisplayed()))
+    }
+
+    @Test
+    fun test_visibility_categories() {
+        //This function tests the visibility of the categories
+        onView(withId(R.id.recyclerView_featuredCategories))
+            .check(matches(isDisplayed()))
+    }
+
+    @Test
+    fun test_navSearchActivity() {
```

Comment:
Test fails with error: `androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: with id: com.sthoray.allright:id/searchActivity`